### DISCUSSION TITLE
using quasar build - all routes 404 Not Found

### DISCUSSION THREAD

**Discussion by vicman1200** [quasarframework/quasar#16170] [General - Components / Directives / etc] [answered]
Hello everyone,

For the first time, something very strange has happened to me. In my **Quasar application**, I have several routes (pages) that work perfectly while I'm developing using `quasar dev`. However, when I perform a `quasar build` only the root of the application is visible. None of the routes or pages can be found, and it always returns a 404 error. I find it curious because everything works fine when using the `quasar dev` command.

Has anyone experienced the same issue? What could be causing this?"

Thank you in advance

```
 "packages": {
    "": {
      "name": "latam-app",
      "version": "0.0.1",
      "dependencies": {
        "@aws-amplify/ui-vue": "^3.1.18",
        "@fontsource/inter": "^5.0.3",
        "@quasar/extras": "^1.0.0",
        "aws-amplify": "^5.2.6",
        "aws-amplify-vue": "^2.1.9",
        "aws-sdk": "^2.1397.0",
        "axios": "^1.2.1",
        "core-js": "^3.6.5",
        "jwt-decode": "^3.1.2",
        "pinia": "^2.0.11",
        "quasar": "^2.6.0",
        "vue": "^3.0.0",
        "vue-router": "^4.0.0"
      },
      "devDependencies": {
        "@babel/eslint-parser": "^7.13.14",
        "@quasar/app-webpack": "^3.0.0",
        "eslint": "^8.10.0",
        "eslint-config-prettier": "^8.1.0",
        "eslint-plugin-vue": "^9.0.0",
        "eslint-webpack-plugin": "^3.1.1",
        "prettier": "^2.5.1"
      }
```
```

**Comment by metalsadman** [comment, marked as answer]
If you're running vue router history mode see https://router.vuejs.org/guide/essentials/history-mode.html#example-server-configurations

**Reply by vicman1200** [reply]
Thank you very much, that's the problem. I'm using History Mode, I applied a rule to catch all to index.html and everything goes good!
